indico/client/client.py

```diff
@@ -47,7 +50,8 @@ def _handle_request_chain(self, chain: RequestChain):
             elif isinstance(request, RequestChain):
                 response = self._handle_request_chain(request)
                 chain.previous = response
-
+            elif isinstance(request, (float, int)):
```

Comment:
this might be... a little brittle if the return type is ever a float or int. Have you thought about using something like a "Backoff" class instead? really simple
```
class Backoff:
  def __init__(self, time):
    self.time = time

return Backoff(5)
```
and then you can
```
if isinstance(req, Backoff):
  sleep(Backoff.time)
```

Reply:
or even just leveraging a function to call like
if isinstance(request, Callable): request()...

Reply:
hell yeah, sounds good - for sleeps, could just... 0 debounce or something yeah - great call